webdriver-spec.html

```diff
@@ -1244,6 +1244,12 @@
   time that commands will block before returning with a <a>timeout</a>
   error.
 
+  <p class=note>Navigation actions - Get, Back, Forward, Refresh - always,
+  unless they abort the navigation, complete with the <a>current top-level browsing context</a>
+  being the <a>current browsing context</a>. Even if we navigate Back or Forward to a state
+  where a different <a href=https://html.spec.whatwg.org/#browsing-context>browsing context</a>
+  was the <a>current browsing context</a>.
```

Comment:
"unless they abort the navigation" is here for the following reason:

Back and Forward commands can complete returning success in certain cases, specifically: "If the task queued by that algorithm aborted before traversing the history, return success with data null."

That step in Back and Forward algorithms, I assume, exists for a reason. But it doesn't allow to assume that a navigation command that returned success performed navigation.

There are certainly ways to make this note more brief. I am happy to amend stuff if there are concrete proposals; I also trust editors just submitting something along the lines with this, without using this PR directly.